### PR TITLE
Update @types

### LINKED ISSUE #503: 'OTSubscriberView' only refers to a type, but is being used as a value here
# Bug Report

**Current behavior**

- Problem with TS and OTSubscriberView

**Steps to reproduce**

- Install v0.17.0 and import OTSubscriberView anywhere

**What is the current *bug* behavior?**

- TS is poping this error: 'OTSubscriberView' only refers to a type, but is being used as a value here.

**What is the expected *correct* behavior?**

- It should not pop this error since OTSubscriberView is a component

**Relevant logs and/or screenshots**

Screenshot 2021-05-14 at 09 13 15

### PR DESCRIPTION
**Pull request by pedrobslisboa**
Incoming changes:
- Improve type names "OTSession as OTSessionProps" instead of "OTSession as OTSessionViewProps" to have both `OTSubscriberViewProps OTSubscriberView` and `OTSubscriberProps and OTSubscriber`
- Add OTSubscriberView export #503

### Contributing checklist
- [X] Code must follow existing styling conventions
- [X] Added a descriptive commit message

### Solves issue(s)
Closes #503

### CODE REVIEW

File: @types/index.d.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ declare module "opentok-react-native" {
     stream: Stream;
   }
 
-  interface OTSessionViewProps extends ViewProps {
+  interface OTSessionProps extends ViewProps {
```
Comment (enricop89) [inline @types/index.d.ts:64]: Can you explain to me why you are renaming the interfaces without the `View` keyword in the name please?

Comment (pedrobslisboa) [reply @types/index.d.ts:64]: IMO it's confusing to have `OTSubscribeViewProps`, it belongs to `OTSubscribeView` or `OTSubscribe`? And it's a common practice to add only the Props to the name of the components to type your Props.

An example from Material UI: https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Accordion/Accordion.d.ts

So I thought it was better to have a direct name related to the component. If the component name is `OTSubscribe` the Props type should be `OTSubscribeProps`.

Comment (enricop89) [reply @types/index.d.ts:64]: That's a good point, thanks for explaining. I will bring it internally and move forward with the PR 😃

Comment (pedrobslisboa) [reply @types/index.d.ts:64]: @enricop89 Any update about it?

**Review by enricop89 — APPROVED** [review]
LGTM

**Pull request merged**